docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -1278,3 +1278,85 @@ function su_humsci_profile_update_9724() {
 
   $config->set('allowed_entity_types.node', $allowed_entity_types)->save();
 }
+
+/**
+ * Clean-up possible duplicates from su_humsci_profile_update_9723().
+ */
+function su_humsci_profile_update_9725(&$sandbox) {
+  $node_type = 'hs_private_page';
+  $results = [];
+  // Remove any dupes from bundle settings.
+  $settings = content_access_get_settings('all', $node_type);
+  foreach ($settings as $operation => &$permissions) {
+    if (!is_array($permissions)) {
+      // This isn't actually an array of perms, this is another setting.
+      continue;
+    }
+    $cleanPermissions = array_unique($permissions);
```

Comment:
```suggestion
    $clean_permissions = array_unique($permissions);
```

Variables should be using `snake_case` here and throughout the rest of the PR to be consistent with the rest of the function/codebase.